components/rmrk/Create/SimpleMint.vue

```diff
@@ -276,12 +276,10 @@ export default class SimpleMint extends mixins(
 
   // handle query results
   public handleResult(data: any) {
-    const {
-      data: {
-        collectionEntities: { nodes },
-      },
-    } = data
-    this.collections = nodes
+    const collectionEntities = data?.data?.collectionEntities
+    if (collectionEntities) {
+      this.collections = collectionEntities.nodes
+    }
```

Comment:
Why?

Reply:
Because at some point when the page loads, the errors below appear
<img width="682" alt="image" src="https://user-images.githubusercontent.com/31397967/158413058-be45aa10-815d-49af-a885-a27c2da7599d.png">

Reply:
By the way, you can also see this error in the issue description.